src/Altinn.AccessManagement.Core/Configuration/KeyVaultSettings.cs

```diff
@@ -0,0 +1,18 @@
+namespace Altinn.AccessManagement.Core.Configuration
+{
+    /// <summary>
+    /// Configuration object used to hold settings for the KeyVault.
+    /// </summary>
+    public class KeyVaultSettings
+    {
+        /// <summary>
+        /// Uri to keyvault
+        /// </summary>
+        public string KeyVaultURI { get; set; }
+
+        /// <summary>
+        /// Name of the certificate secret
+        /// </summary>
+        public string PlatformCertSecretId { get; set; } = "platform-access-token-private-cert";
```

Comment:
I belive the value in platform is different for this cert. Remi is working on a pull request that also is adding this. So whoever merge last needs to fix a merge conflig. Check with other platform component what value is in platform.